plugins/mcreator-localization/lang/texts.properties

```
@@ -1202,6 +1202,8 @@ blockly.block.text_format_number.format=Format number:
 blockly.block.text_format_number.as=as:
 blockly.block.time_to_formatted_string=Time as formatted text
 blockly.block.return=Return
+blockly.block.entity_data_get=Get the data parameter of the custom entity %1 
+blockly.block.entity_data_set=Set the data parameter of the custom entity %1 to %2 
```

I'd correct the wording a bit so that it doesn't stack all inputs and fields at the same place:
```suggestion
blockly.block.entity_data_get=Get value of custom entity %1 data parameter 
blockly.block.entity_data_set=Set to %2 value of custom entity %1 data parameter 
```